app/services/integrations/aggregator/payments/create_service.rb

```diff
@@ -38,7 +46,7 @@ def call
 
           deliver_error_webhook(customer:, code:, message:)
 
-          raise e
+          raise e if e.error_code.to_i >= 500
```

Comment:
Maybe we should make sure to return a result (empty, I guess). What do you think?
```suggestion
          return result if e.error_code.to_i < 500
          
          raise e
```